openapi/p4pa-pu-bff.openapi.yaml

```
@@ -47,6 +47,22 @@ paths:
           description: Forbidden - Access denied
         '500':
           description: Internal Server Error
+  /health:
+    get:
+      tags:
+        - Monitoring
+      operationId: getHealthStatus
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceStatus'
+        '500':
+          description: Internal Server Error
```

same body type as 200
```suggestion
        '503':
          description: Internal Server Error
```